### PR TITLE
fix: only save on blur textarea

### PR DESCRIPTION
**Pull request by niekvanstaveren**
Mostly prettier changes. Real changes are here:
https://github.com/qlik-oss/qsSimpleKPI/pull/107/files?diff=split&w=1#diff-c692b18c3cfdb36adf36f3a672d2650e5020fe52e455471e1dd232aff44ec2abR106
https://github.com/qlik-oss/qsSimpleKPI/pull/107/files?diff=split&w=1#diff-c692b18c3cfdb36adf36f3a672d2650e5020fe52e455471e1dd232aff44ec2abR132-R137

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,20 +12,20 @@ defaults: &defaults
...
           command: npm install
-      - run:
-          name: BlackDuck scan
-          command: curl -s https://detect.synopsys.com/detect.sh | bash -s -- \
-            --blackduck.url="https://qliktech.blackducksoftware.com" \
-            --blackduck.trust.cert=true \
-            --blackduck.username="svc-blackduck" \
-            --blackduck.password=${svc_blackduck} \
-            --detect.project.name="viz-bundle-qlik-multi-kpi"
+      # - run:
```
Comment (niekvanstaveren) [inline .circleci/config.yml:21]: This fails on Circle. Not sure how to fix it.

Comment (haxxmaxx) [reply .circleci/config.yml:21]: what does it do?

Comment (niekvanstaveren) [reply .circleci/config.yml:21]: Black Duck allows you to scan applications and container images, identify all open source components, and detect any open source security vulnerabilities, compliance issues, or code-quality risks.

We removed it from client a long time ago: https://github.com/qlik-trial/sense-client/commit/02bed33397c7ff30cd7bffb734782f7d1390cdfd

I guess something that Dependabot does currently. I'll see if it is setup here

Comment (niekvanstaveren) [reply .circleci/config.yml:21]: I have enabled renovate now and also have dependabot check dependencies and create PRs for them (see PR list 😄 )

Comment (haxxmaxx) [reply .circleci/config.yml:21]: then we might as well remove this code all together

**Review by haxxmaxx — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (niekvanstaveren) [reply .circleci/config.yml:21]: Yes, probably better indeed

**Pull request merged**